doc/manual/index.adoc

```diff
@@ -236,20 +315,171 @@ a custom resolver.
 
 == Writing Query Resolvers
 
-(TBD)
+In stillsuit we use the term "query resolver" to refer to a lacinia resolver that returns one
+or more entities the provide the basic data for a GraphQL query.
+
+Basically a query is a regular resolver function that will return one or more
+https://docs.datomic.com/on-prem/entities.html[datomic entity objects].
+
+Here's an example of writing a query resolver. First let's define the datomic schema we'll
+be working with.
+
+First we'll define the lacinia schema for the query. For this example, we'll define a very simple
```

Comment:
You repeat "First" here. I think you can be more concise here in the setup and let the code do the talking.

Reply:
I tightened up the prose some.